extract/extract.py

```diff
@@ -1189,9 +1189,16 @@ def extract_script(self, request: ServiceRequest, local: str, encoding: str):
         scripts = soup.findAll("script")
         extracted = []
         for script in scripts:
-            if script.string is None:
+            # Make sure there is actually a body to the script
+            body = script.string
+            if body is None:
                 continue
-            encoded_script = str(script.string).encode()
+            body = str(body).strip()  # Remove whitespace
+            if len(body) == 0:
```

Comment:
This can be simplified to `if body:`

Reply:
I was thinking we might want it to actually be `len(body) < X` but I wasn't sure what a safe X for a script might be.